cmd/scollector/collectors/processes_linux.go

```diff
@@ -91,6 +97,8 @@ func linuxProcMonitor(w *WatchedProc, md *opentsdb.MultiDataPoint) error {
 				}
 			}
 		}
+		start_ts := file_status.ModTime().Unix()
+		now := time.Now().Unix()
```

Comment:
Just use `now()` for this: https://github.com/bosun-monitor/bosun/blob/3e9002eba1699fe4db468ae4c7cc0aee06e96842/cmd/scollector/collectors/collectors.go#L107